src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -157,45 +157,42 @@ public class SendReceiveTest : ClientWebSocketTestBase
                 var recvBuffer = new byte[100];
                 var recvSegment = new ArraySegment<byte>(recvBuffer);
 
-                try
+                for (int i = 0; i < tasks.Length; i++)
                 {
-                    for (int i = 0; i < tasks.Length; i++)
-                    {
-                        tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
-                    }
+                    tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
+                }
 
-                    Task.WaitAll(tasks);
+                Exception e = Record.Exception(() => Task.WaitAll(tasks));
```

Comment:
Note, also, that `Assert.Throws` _can't_ be used here, since there is no guarantee that an exception will actually be thrown (presumably based on other parameters, although I didn't look terribly far into these tests at the moment).